app/code/community/Sendsmaily/Sync/Model/Cron.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * Sendsmaily Sync
+ * Export Magento newsletter subscribers to Sendsmaily
+ * Copyright (C) 2010 Sendsmaily
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+class Sendsmaily_Sync_Model_Cron
+{
+    /**
+     * Limit Smaily unsubscribers request size.
+     */
+    const UNSUBSCRIBE_BATCH_LIMIT = 1000;
+
+    /**
+     * Cron task to synchronize Sendsmaily database with Magento
+     * newsletter subscribers.
+     *
+     * @return void
+     */
+    public function customerSync()
+    {
+        // Has module been activated? Is sync enabled?
+        if (Mage::getStoreConfig('newsletter/sendsmaily/active') == false ||
+            Mage::getStoreConfig('newsletter/sendsmaily/active_sync') == false) {
+        return;
```

Comment:
Formatting: indentation. `return` should be one more step to the right.

Reply:
Done 608fbd51c1b00c0b7b2dfcd11dda1b068a5c5e16